test/parallel/test-stream-iterator-helpers-test262-tests.mjs

```diff
@@ -0,0 +1,115 @@
+import '../common/index.mjs';
+import { Readable } from 'stream';
+import assert from 'assert';
+
+// These tests are manually ported from the draft PR for the test262 test suite
+// Authored by Rick Waldron in https://github.com/tc39/test262/pull/2818/files
+
+// test262 license:
```

Comment:
This arguably should be added to LICENSE instead with a comment here indicating that's where the relevant license info is located. But not blocking.